### DISCUSSION TITLE
2024 G14:  Keyboard backlight timeout when using Windows Dynamic Lighting

### DISCUSSION THREAD

**Discussion by Noisrevid** [seerge/g-helper#2170] [Q&A]
It's me again @seerge !      Are you using Windows Dynamic lighting with Ghelper (for your 2024 G14) or just Ghelper to control only?    I'd prefer using Windows Dynamic Lighting as it has a super granule way to dim the lighting very precisely.     So my issue is that Windows Dynamic Lighting does not offer a keyboard lighting time-out function..   So I enabled it to 30 seconds in Ghelper and ultimately it eventually works but at the 30sec mark the keyboard flashes off and then back on..  and the next 30 seconds it finally turns the keyboard off..  So 30 seconds turns into 1 min.. It seems to be buggy when it attempts to turn off the lighting at 30 seconds the first time.    I could probably "cheese" this by setting it to 15 seconds in Ghelper to make it so that it ends up being 30 seconds.

Thanks,

Div

**Comment by seerge** [comment]
I advice to turn Dynamic Lightning off, as it will interfere.

**Reply by Noisrevid** [reply]
10-4..   seems like the optimal solution, was just curious, thanks again.
